packages/component/src/Middleware/Attachment/createCoreMiddleware.js

```diff
@@ -20,8 +20,8 @@ export default function createCoreMiddleware() {
           attachment: { contentType, contentUrl, thumbnailUrl } = {}
         }
       ] = args;
-
-      return role === 'user' && !/^text\//u.test(contentType) && !thumbnailUrl ? (
+      // && !attachment.content
+      return role === 'user' && !/text\//u.test(contentType) && !thumbnailUrl ? (
```

Comment:
Why removing the `^` (starts with operator)?

Reply:
this was a typo. fixing.